### DISCUSSION TITLE
I need help with installing (compiling)

### DISCUSSION THREAD

**Discussion by OlekAleksander** [IrisShaders/Iris#96] [Q&A] [answered]
I know i need to compile this, but i dont know how.
Can someone send me compiled mod (sodium version).

Thanks.
(sorry for bad english i`m from poland)

**Comment by TheKodeToad** [comment]
https://github.com/IrisShaders/Iris/suites/1929115487/artifacts/38037976

**Reply by turtIeboba** [reply]
there's nothing there?

**Reply by K0-RR** [reply]
What do you mean by nothing? Yes, the site is empty but that’s always the case with direct download links. I checked and it is downloading but this version probably is already outdated.

**Comment by K0-RR** [comment, marked as answer]
https://github.com/IrisShaders/Iris/discussions/67
